types/tpdirect/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for non-npm package tpdirect-browser 1.0
```

Comment:
The readme suggests that these are for version 5.5.

Reply:
Yes, it is. Does that mean it must be same as current SDK version 5.5?
I thought it is the major version of this typing instead of SDK version.

Reply:
I changed it to `tpdirect-browser 5.5`

Reply:
Thanks!  Some notes on type versions: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Thank for review!